packages/env/lib/env.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+const dockerCompose = require( 'docker-compose' );
+const NodeGit = require( 'nodegit' );
+const wait = require( 'util' ).promisify( setTimeout );
+
+/**
+ * Internal dependencies
+ */
+const createDockerComposeConfig = require( './create-docker-compose-config' );
+
+// Config Variables
+const pluginPath = process.cwd();
+const pluginName = path.basename( pluginPath );
+const pluginTestsPath = fs.existsSync( './packages' ) ? '/packages' : '';
+const dockerComposeOptions = {
+	config: path.join( __dirname, 'docker-compose.yml' ),
+};
+
+// WP CLI Utils
+const wpCliRun = ( command, isTests = false ) =>
+	dockerCompose.run(
+		`${ isTests ? 'tests-' : '' }wordpress-cli`,
+		command,
+		dockerComposeOptions
+	);
+const setupSite = ( isTests = false ) =>
+	wpCliRun(
+		`wp core install --url=localhost:888${
```

Comment:
It probably shouldn't be hardcoded to two values: 8888 and 8889. In addition, the title should be based on the name of the plugin or theme.

Reply:
Yeah, that's already changed in the first follow up: #17697